<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QDialog" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255)</string>
  </property>
  <widget class="QLabel" name="lblAboutUs">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>About us</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>We are                            and </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>two up-and-coming developers working for                               .</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string> is our first common project which was assigned to us</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>by our customer                        (BWS).</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string> Adrian Werner</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Deutsche Börse AG</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HangMe™ </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Peter Grüning</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enjoy our Game!</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCopyright">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>© 2016 ALEXANDER HÖHN &amp; ADRIAN WERNER ALL RIGHTS RESERVED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEasterEgg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(x, x, x, 10)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Alexander Höhn</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAboutUs_12">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <zorder>lblAboutUs</zorder>
  <zorder>lblAboutUs_10</zorder>
  <zorder>lblCopyright</zorder>
  <zorder>btnEasterEgg</zorder>
  <zorder>lblAboutUs_5</zorder>
  <zorder>lblAboutUs_9</zorder>
  <zorder>lblAboutUs_8</zorder>
  <zorder>lblAboutUs_4</zorder>
  <zorder>lblAboutUs_3</zorder>
  <zorder>lblAboutUs_7</zorder>
  <zorder>lblAboutUs_2</zorder>
  <zorder>lblAboutUs_6</zorder>
  <zorder>lblAboutUs_11</zorder>
  <zorder>lblAboutUs_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
